emails/welcome-email.tsx

```diff
@@ -35,7 +35,7 @@ export const WelcomeEmail = ({ userFirstname }: WelcomeEmailProps) => (
           alt="iShortn"
           style={logo}
         />
-        <Text style={paragraph}>Hi {userFirstname},</Text>
+        <Text style={paragraph}>Hi {userFirstname !== null ? userFirstname : 'there'},</Text>
```

Comment:
can we simplify the statement to something like `userFirstName ? userFirstName : 'there'`

Reply:
Sure, Done!